packages/README.md

```diff
@@ -0,0 +1,4 @@
+# External packages
+
+This directory contains the source code of packages that we need, but aren't available in the
+distribution that we use, so we build them from source.
```

Comment:
Can we list where we get these from for when we need to upgrade?

Reply:
I moved the code to build the packages to separate `build.sh` scripts inside the `packages` directory. For example, for `xz` it is now in `packages/xz/bulid.sh`. Those scripts contain a `version` environment variable that can be used to update the package. I tried to document it in the new version of this README.md.